src/components/AutoComplete/AutoComplete.js

```diff
@@ -95,7 +95,9 @@ export default class AutoComplete extends Component {
 
   render() {
     const { value } = this.state;
-    const { id, source } = this.props;
+    const { source } = this.props;
+    let { id } = this.props;
+    id = `${id}-${Math.random()}`;
```

Comment:
Sorry. Is this relevant to `prevent autocomplete`?
Also, why indeed you need to generate random id (and if so, why the component is doing this instead of the component which using the `AutoComplete`?

And last note, if you're override the `id` variable, why do you need to declare it (using destruction) from the first place?

Instead of
```
let { id } = this.props;
id = `${id}-${Math.random()}`;
```

Just

```
id = `${id}-${Math.random()}`;
```